### DISCUSSION TITLE
Handler :: UpdateCommand - Atualização de apenas alguns atributos específicos

### DISCUSSION THREAD

**Discussion by lucasapoena** [balta-io/7196#3] [Perguntas e Respostas] [answered]
Balta, 

Caso eu deseje atualizar apenas alguns atributos específicos de uma entidade, como que ficaria o mapeamento destes atributos? 
No exemplo do projeto a entidade possui apenas o campo _Title_.
Caso possuíssemos  também outro campo, por exemplo _Description_, e quando fosse realizado o processo de atualização dos valores da entidade, eu desejasse atualizar apenas este campo especifico, como que poderia ser realizado?

Atualmente com o exemplo informado, necessito mapear manualmente os campos e no caso os valores sempre são setados pelos valores provenientes do command.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @lucasapoena, como vai?

Não sei se entendi muito bem, mas caso fosse atualizar algo específico, você poderia ter um command para isto.
Teria por exemplo: UpdateEmailCommand, UpdateUserInfoCommand.

No meu caso mesmo (Aqui no balta), como temos muitas relações via E-mail, temos dois comandos, um que atualiza nome, título, bio e outras coisas bobas, e outro que atualiza apenas o E-mail.

Sei que são parecidos, mas as razões deles existirem são bem distintas... O processo de alterar E-mail por exemplo faz diversas coisas, então não caberia tudo no mesmo comando de atualizar as informações básicas!

Era isto?

**Reply by andrebaltieri** [reply]
@lucasapoena posso marcar como respondido?

**Reply by lucasapoena** [reply]
Opa Balta, entendi sim. Pode fechar :)
